src/main/resources/xsl/sarif.xsl

```diff
@@ -432,10 +426,10 @@
         <xsl:text>" } }</xsl:text>
     </xsl:template>
 
-    <xsl:template name="get_matched_project_path">
+    <xsl:template name="get_matching_project_path">
         <!--   Select the matched project path from tempProjectRootPathElements by uri attribute    -->
-        <xsl:param name="nodeWithUriAttribute"/>
-        <xsl:variable name="defaultProjectPath" select="$tempProjectRootPathElements/PROJECTROOT[starts-with($nodeWithUriAttribute/@uri, @uri)]"/>
+        <xsl:param name="checkedFileWithUri"/>
```

Comment:
My only question is about this parameter's name. We used this template in two places: **artifactLocation** and **artifacts**. I don't think there is a concept called **checkedFile** in **artifactLocation**. What if we only pass **@uri** to this template instead of a Node?